tests/test_root_commands.py

```diff
@@ -131,3 +131,19 @@ def complete(partialPath):
     assert len(complete('monitors.focus.')) >= 8
     assert complete('t') == ['tags.', 'theme.', 'tmp.']
     assert complete('') == [l + '.' for l in hlwm.list_children_via_attr('')]
+
+@pytest.mark.parametrize('attrtype', ['int', 'bool', 'string', 'color', 'uint'])
+@pytest.mark.parametrize('name', ['my_test', 'my_foo'])
```

Comment:
It's not really related to this PR, but another simple but good test case you could add for `new_attr` is testing its behavior when the attribute name doesn't start with `my_`.